### DISCUSSION TITLE
Bootstrap returning no such host

### DISCUSSION THREAD

**Discussion by jtgorny** [fluxcd/flux2#2604] [Q&A] [answered]
# Bootstrap
flux bootstrap gitlab \
--hostname=host \
--owner=gornyj \
--repository=flux-cd \
--branch=master \
--path=./k8s/ \
--token-auth \
--namespace=flux \
--personal

# Output
► connecting to https://host
✗ failed to get Git repository "https://host/gornyj/flux": Get "https://host/api/v4/projects/gornyj%2Fflux": dial tcp: lookup host on [host]:53: no such host

# Noteworthies
Cluster CoreDNS is configured properly. Local resolv.conf is also configured with the proper nameserver for this private gitlab address. CA is trusted.

**Comment by kingdonb** [comment, marked as answer]
I am not sure how CoreDNS has been configured, but if you set `/etc/resolv.conf` on your local machine it will not advertise this hostname in CoreDNS.

Can you say more about how you configured CoreDNS? Here is an example we had presented at the Flux Bug Scrub a few weeks ago where CoreDNS is configured for a local git server in like-airgap environment:

https://github.com/fragolinux/k3d-local-git/blob/de7111e27a197e42f0305b508cdcc3f5eeb6b4ec/k3d_local_git.sh#L114-L130

Here it is presented in context:

https://www.youtube.com/watch?v=hNt3v0kk6ec

If this helps or not please let us know! We will need more information about your config in order to help further.
